package.json

```diff
@@ -1,17 +1,17 @@
 {
-  "name": "@servall/app-config",
+  "name": "@ldev/app-config",
```

Comment:
Seeing `ldev` in many places instead of `lcdev`

Reply:
Oops, fixed